voucher/src/main/java/com/adyen/checkout/voucher/internal/ui/view/FullVoucherView.kt

```diff
@@ -186,6 +193,35 @@ internal class FullVoucherView @JvmOverloads constructor(
         }
     }
 
+    private fun updateReadInstructionTextView(instructionUrl: String?) {
+        binding.textViewReadInstructions.isVisible = instructionUrl != null
+        instructionUrl?.let {
+            binding.textViewReadInstructions.text =
+                localizedContext.getString(R.string.checkout_voucher_read_instructions).formatFullStringWithHyperLink()
+            binding.textViewReadInstructions.setOnClickListener {
+                onReadInstructionsClicked(instructionUrl)
+            }
+        }
+    }
+
+    private fun onReadInstructionsClicked(url: String) {
+        val defaultColors = CustomTabColorSchemeParams.Builder()
+            .setToolbarColor(ThemeUtil.getPrimaryThemeColor(context))
+            .build()
+
+        try {
+            CustomTabsIntent.Builder()
+                .setShowTitle(true)
+                .setDefaultColorSchemeParams(defaultColors)
+                .build()
+                .launchUrl(context, Uri.parse(url))
+
+            adyenLog(AdyenLogLevel.DEBUG) { "Successfully opened pdf in custom tab" }
```

Comment:
If this is not always a PDF, could you change the logs here?

Reply:
My bad to leave the logs after copy-paste, it's not a pdf link.